Content/Items/Vitric/Weapons.MagmiteVacpack.cs

```diff
@@ -439,7 +429,9 @@ public override void ModifyHitByProjectile(NPC npc, Projectile projectile, ref i
         {
             Player player = Main.player[projectile.owner];
 
-            if (projectile.owner == MagmiteOwner && projectile.friendly && (projectile.minion || projectile.DamageType == DamageClass.Summon || ProjectileID.Sets.MinionShot[projectile.type] == true) && npc.whoAmI == player.MinionAttackTargetNPC && MagmiteAmount > 0 && player.HasMinionAttackTargetNPC)
+            bool IsMinion = (projectile.minion || projectile.DamageType == DamageClass.Summon || ProjectileID.Sets.MinionShot[projectile.type] == true);
```

Comment:
This name seems misleading, why would it be a minion if its a minion or a summon projectile? (whips are summon projectiles!)

Reply:
by IsMinion I just meant that its a minion damage type, I should have, and will rename to bool to IsSummon. Can you elaborate on the wrong conditions that Im checking?

Reply:
I was mostly reffering to detecting summoner projectiles in general as being potentially incorrect. If thats intentional than you are fine I believe.

Reply:
Yes I was intending to detect all summoner projectiles in general, I should have named the bool more clearly though, thats my bad. Will be fixed asap